About

danielz117 opened the pull request:
Completed the About Us page based on the Figma. The dimensions on the Figma were slightly less wide than a webpage so I tried my best to stretch content to fill space.

bdubs006 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.css

danielz117 committed:
Update About.css

danielz117 committed:
Update About.css

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Add files via upload

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Add files via upload

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Add files via upload

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

danielz117 committed:
Update About.jsx

chauandrew commented on client/src/pages/About/About.jsx:
```diff
@@ -1,12 +1,71 @@
 import React from 'react'
 import './About.css'
+import flowers3 from '../../assets/Flowers3.png'
+import flowers2 from '../../assets/Flowers2.png'
+import treesBig from '../../assets/TreesBig.png'
+import treesSmall from '../../assets/TreesSmall.png'
+import ladyLaptop from '../../assets/LadyLaptop.png'
+import manLaptop from '../../assets/ManLaptop.png'
```
Can we put these all in a subdirectory? eg, `assets/about/Flowers3`, and so on.

bdubs006 replied:
Updated images to about_assets folder under assets

chauandrew commented on client/src/pages/About/About.jsx:
```diff
@@ -1,12 +1,71 @@
+            <img src={manLaptop} alt = "man laptop"
+                style = {{position: 'relative', left: 505, top: 291, height: '12%', width: '12%'}}
+            />
+            <style>
+                @import url('https://fonts.googleapis.com/css2?family=DM+Sans&display=swap');
+            </style>
```
Can you move this line to `/client/src/App.css`? That's where global css lives, and chances are that this font will be used on a lot of other pages too.

bdubs006 replied:
Updated!

chauandrew commented on client/src/pages/About/About.jsx:
```diff
@@ -1,12 +1,71 @@
+import treesBig from '../../assets/TreesBig.png'
+import treesSmall from '../../assets/TreesSmall.png'
+import ladyLaptop from '../../assets/LadyLaptop.png'
+import manLaptop from '../../assets/ManLaptop.png'
 
 const About = () => {
     return (
         <div className="page-content">
-            <h1>About us</h1>
+            <img src={flowers3} alt = "three flowers" 
+                style = {{position: 'relative', left: 260, top: 113, height: '3.187%', width: '3.187%'}}
+            />
```
Don't worry too much about it, but in the future, it's better practice to add classes to your html and then put your css in the `About.css` file, rather than adding styles straight to the code. It makes it much easier to reuse code.

bdubs006 replied:
updated css and pushed to master!

chauandrew reviewed (approved):
The page looks really good! I left a few comments, but feel free to merge your code! 

This may be pretty challenging, but a good next step would be to work towards making the design responsive (eg making sure it looks good on narrower pages).

bdubs006 committed:
About page contents

basic frame of how content should be

bdubs006 committed:
Merge branch 'About' of https://github.com/chauandrew/gp-edu into About

bdubs006 committed:
About page comments

1. moved assets for about page into subdirectory folder.
2. moved font import url to App.css

bdubs006 committed:
About styling reorganization

edited styling to be formatted in .css file
